app.py

```diff
@@ -0,0 +1,139 @@
+from flask import Flask, request, abort
+from git import Repo
+import ansible_runner
+import os
+import requests
+import json
+import configparser
+
+from datetime import datetime
+
+app = Flask(__name__)
+
+def appendTime(message):
+    output = "[" + datetime.utcnow().isoformat() + "]" + " " + message
+    return (output)
+
+def updateRepo(repoName, repoURL): #Update the local git repo
```

Comment:
Something else to think about: in this function, you're always applying changes from the `HEAD` of the repository, rather than using the commit included in the push notification from GitHub. This is *probably* fine, but it means that there's a chance you will changing the configuration of switches that aren't mentioned in the notification sent to slack (e.g., if a second commit is pushed to the repository between the time you receive the push notification from GitHub and the time you update the local copy of the repository).